src/common/segment-utils.ts

```diff
@@ -104,3 +105,63 @@ type FacebookBasicEvent =
 export const trackFacebookBasicEvent = (eventName: FacebookBasicEvent) => {
   analytics.track(eventName, {}, { integrations: { All: false, 'Facebook Pixel': true } });
 };
+
+/**
+ * Register a Segment plugin that will parse url paths from learn.xip.co.
+ *
+ * @see https://github.com/segmentio/analytics-next/tree/master/packages/browser#-plugins
+ *
+ */
+export const urlPathFilter: Plugin = {
+  name: 'Clickfunnels Path Filter',
+  type: 'enrichment',
+  version: '1.0.0',
+
+  isLoaded: () => true,
+  load: () => Promise.resolve(),
+
+  track: (ctx) => {
+    const url = new URL(ctx.event.properties?.url || '');
+    if (url.hostname !== 'learn.xip.co') {
```

Comment:
Since I added the plugin to clickfunnels/segment-utils.js I don't think there's any reason to restrict anymore. This should only be called when a clickfunnels url exists anyways.